components/layout/Header.tsx

```diff
@@ -11,14 +11,22 @@ const HeaderWrapper = styled.nav`
   flex-direction: column;
   align-items: center;
   gap: 1rem;
-  padding: 0.75rem 1.25rem;
+  padding: 2rem 1.25rem 1rem;
```

Comment:
```suggestion
  padding: 1.5rem 1.25rem 1rem; 
```

p2) 이부분만 이렇게 바꿔야 되는거 아닌가? 
![image](https://github.com/cancook/cancook-frontend/assets/49224104/612dfed0-9941-42c1-ab13-964756814fe3)
24px => 1.5rem으로 알고있어